_package/pyproject.toml

```
@@ -8,7 +8,7 @@ name = "pyconverter-generatedcommands"
 version = "0.1.dev0"
 description = "Documentation."
 readme = "README.rst"
-requires-python = ">=3.7"
+requires-python = ">=3.10,<3.13"
```

```suggestion
requires-python = ">=3.10,<4"
```
I think it's best if we just limit the major version